app/assets/javascripts/templates/story.ejs

```diff
@@ -49,7 +49,7 @@
       <% }); %>
     </span>
   <% } %>
-  <%= story.escape('title') %>
+  <%- story.escape('title') %>
```

Comment:
Yes, it will just avoid escaping twice, since `story.escape` already escapes the property.